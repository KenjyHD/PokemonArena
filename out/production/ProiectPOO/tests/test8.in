Nume|Varsta|NumePokemon1;NumePokemon2;NumePokemon3|Obiect1Pokemon1;Obiect2Pokemon1;Obiect3Pokemon1|Obiect1Pokemon2;Obiect2Pokemon2;Obiect3Pokemon2|Obiect1Pokemon3;Obiect2Pokemon3;Obiect3Pokemon3
Marcel|28|Squirtle;Pikachu;Meowth|Vitamine;Scut;Baghetă Magică|Vitamine;Brad de Crăciun;Pelerină|Vestă;Pelerină;Vitamine
Robert|29|Squirtle;Psyduck;Bulbasaur|Brad de Crăciun;Vestă;Săbiuță|Vestă;Baghetă Magică;Brad de Crăciun|Brad de Crăciun;Săbiuță;Baghetă Magică