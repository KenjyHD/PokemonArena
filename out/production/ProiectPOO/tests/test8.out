****** In arena intra Marcel si Robert ******
###########################################
###############  Luptele 1  ###############
###########################################

------ Se lupta Meowth cu Neutrel2 ------
Meowth {HP[41], AN[3], Def[4], SDef[2]}
Neutrel2 {HP[20], AN[4], Def[1], SDef[1]}
--- Caracteristici cu obiecte echipate ---
Meowth {HP[53], AN[5], Def[4], SDef[5]}
Neutrel2 {HP[20], AN[4], Def[1], SDef[1]}

Runda 1
Meowth HP [41]
CDs: Abilitate1 [0] / Abilitate2 [0]

Neutrel2 HP [20]

Meowth foloseste atac normal -> 4
Neutrel2 foloseste atac normal -> 0

Runda 2
Meowth HP [41]
CDs: Abilitate1 [0] / Abilitate2 [0]

Neutrel2 HP [16]

Meowth foloseste abilitate1 -> 5
Neutrel2 foloseste atac normal -> 0

Runda 3
Meowth HP [41]
CDs: Abilitate1 [4] / Abilitate2 [0]
Dodged 

Neutrel2 HP [11]

Meowth foloseste atac normal -> 4
Neutrel2 foloseste atac normal -> 0

Runda 4
Meowth HP [41]
CDs: Abilitate1 [3] / Abilitate2 [0]

Neutrel2 HP [7]

Meowth foloseste abilitate2 -> 1
Neutrel2 foloseste atac normal -> 0

Runda 5
Meowth HP [41]
CDs: Abilitate1 [2] / Abilitate2 [3]
Dodged 

Neutrel2 HP [6]

Meowth foloseste atac normal -> 4
Neutrel2 foloseste atac normal -> 0

Runda 6
Meowth HP [41]
CDs: Abilitate1 [1] / Abilitate2 [2]

Neutrel2 HP [2]

Meowth foloseste atac normal -> 4
Neutrel2 foloseste atac normal -> 0

---------> A castigat Meowth <---------
Meowth {HP[41], AN[3], Def[4], SDef[2]}
				  V
Meowth {HP[42], AN[4], Def[5], SDef[3]}


------ Se lupta Bulbasaur cu Neutrel2 ------
Bulbasaur {HP[42], AS[5], Def[3], SDef[1]}
Neutrel2 {HP[20], AN[4], Def[1], SDef[1]}
--- Caracteristici cu obiecte echipate ---
Bulbasaur {HP[42], AS[8], Def[4], SDef[1]}
Neutrel2 {HP[20], AN[4], Def[1], SDef[1]}

Runda 1
Bulbasaur HP [42]
CDs: Abilitate1 [0] / Abilitate2 [0]

Neutrel2 HP [20]

Bulbasaur foloseste atac special -> 7
Neutrel2 foloseste atac normal -> 0

Runda 2
Bulbasaur HP [42]
CDs: Abilitate1 [0] / Abilitate2 [0]

Neutrel2 HP [13]

Bulbasaur foloseste abilitate1 -> 6
Neutrel2 foloseste atac normal -> 0

Runda 3
Bulbasaur HP [42]
CDs: Abilitate1 [4] / Abilitate2 [0]

Neutrel2 HP [7]

Bulbasaur foloseste abilitate2 -> 5
Neutrel2 foloseste atac normal -> 0

Runda 4
Bulbasaur HP [42]
CDs: Abilitate1 [3] / Abilitate2 [3]

Neutrel2 HP [2]

Bulbasaur foloseste atac special -> 7
Neutrel2 foloseste atac normal -> 0

---------> A castigat Bulbasaur <---------
Bulbasaur {HP[42], AS[5], Def[3], SDef[1]}
				  V
Bulbasaur {HP[43], AS[6], Def[4], SDef[2]}

############################################
###########  Luptele semifinale  ###########
############################################

------ Se lupta Squirtle cu Squirtle ------
Squirtle {HP[60], AS[3], Def[5], SDef[5]}
Squirtle {HP[60], AS[3], Def[5], SDef[5]}
--- Caracteristici cu obiecte echipate ---
Squirtle {HP[62], AS[8], Def[7], SDef[7]}
Squirtle {HP[70], AS[3], Def[6], SDef[5]}

Runda 1
Squirtle HP [60]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [60]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste abilitate2 -> 2

Runda 2
Squirtle HP [58]
CDs: Abilitate1 [0] / Abilitate2 [2]
Stunned

Squirtle HP [58]
CDs: Abilitate1 [0] / Abilitate2 [2]
Stunned

Squirtle este stunned
Squirtle este stunned

Runda 3
Squirtle HP [58]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle HP [58]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle foloseste atac special -> 3
Squirtle foloseste abilitate1 -> 4

Runda 4
Squirtle HP [54]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [55]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle foloseste atac special -> 3
Squirtle foloseste atac special -> 0

Runda 5
Squirtle HP [54]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [52]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle foloseste atac special -> 3
Squirtle foloseste atac special -> 0

Runda 6
Squirtle HP [54]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [49]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle foloseste atac special -> 3
Squirtle foloseste abilitate2 -> 2

Runda 7
Squirtle HP [52]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Squirtle HP [46]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste abilitate1 -> 4

Runda 8
Squirtle HP [48]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [46]
CDs: Abilitate1 [3] / Abilitate2 [1]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 9
Squirtle HP [48]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle HP [44]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste abilitate1 -> 4

Runda 10
Squirtle HP [48]
CDs: Abilitate1 [3] / Abilitate2 [1]

Squirtle HP [40]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle foloseste atac special -> 3
Squirtle foloseste abilitate2 -> 2

Runda 11
Squirtle HP [46]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Squirtle HP [37]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste abilitate1 -> 4

Runda 12
Squirtle HP [42]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle HP [37]
CDs: Abilitate1 [3] / Abilitate2 [1]

Squirtle foloseste atac special -> 3
Squirtle foloseste atac special -> 0

Runda 13
Squirtle HP [42]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [34]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle foloseste abilitate1 -> 4
Squirtle foloseste abilitate2 -> 2

Runda 14
Squirtle HP [40]
CDs: Abilitate1 [3] / Abilitate2 [0]
Stunned

Squirtle HP [30]
CDs: Abilitate1 [1] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste atac special -> 0

Runda 15
Squirtle HP [40]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle HP [30]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle foloseste atac special -> 3
Squirtle foloseste abilitate1 -> 4

Runda 16
Squirtle HP [36]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle HP [27]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle foloseste atac special -> 3
Squirtle foloseste atac special -> 0

Runda 17
Squirtle HP [36]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [24]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle foloseste abilitate1 -> 4
Squirtle foloseste abilitate2 -> 2

Runda 18
Squirtle HP [34]
CDs: Abilitate1 [3] / Abilitate2 [0]
Stunned

Squirtle HP [20]
CDs: Abilitate1 [1] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste atac special -> 0

Runda 19
Squirtle HP [34]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle HP [20]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 20
Squirtle HP [34]
CDs: Abilitate1 [1] / Abilitate2 [2]

Squirtle HP [18]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste atac special -> 3

Runda 21
Squirtle HP [34]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle HP [15]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste atac special -> 3
Squirtle foloseste abilitate2 -> 2

Runda 22
Squirtle HP [32]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Squirtle HP [12]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste atac special -> 0

Runda 23
Squirtle HP [32]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [12]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle foloseste atac special -> 3
Squirtle foloseste abilitate1 -> 4

Runda 24
Squirtle HP [28]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [9]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle foloseste abilitate1 -> 4
Squirtle foloseste atac special -> 0

Runda 25
Squirtle HP [28]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle HP [5]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 26
Squirtle HP [28]
CDs: Abilitate1 [2] / Abilitate2 [2]

Squirtle HP [3]
CDs: Abilitate1 [1] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste atac special -> 3

---------> A castigat Squirtle <---------
Squirtle {HP[60], AS[3], Def[5], SDef[5]}
				  V
Squirtle {HP[61], AS[4], Def[6], SDef[6]}


------ Se lupta Pikachu cu Psyduck ------
Pikachu {HP[35], AS[4], Def[2], SDef[3]}
Psyduck {HP[43], AN[3], Def[3], SDef[3]}
--- Caracteristici cu obiecte echipate ---
Pikachu {HP[37], AS[6], Def[3], SDef[6]}
Psyduck {HP[53], AN[6], Def[4], SDef[3]}

Runda 1
Pikachu HP [35]
CDs: Abilitate1 [0] / Abilitate2 [0]

Psyduck HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Pikachu foloseste abilitate2 -> 4
Psyduck foloseste abilitate1 -> 2

Runda 2
Pikachu HP [35]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Psyduck HP [39]
CDs: Abilitate1 [4] / Abilitate2 [0]
Stunned

Psyduck este stunned
Pikachu foloseste abilitate1 -> 6

Runda 3
Pikachu HP [35]
CDs: Abilitate1 [4] / Abilitate2 [4]

Psyduck HP [33]
CDs: Abilitate1 [3] / Abilitate2 [0]

Pikachu foloseste atac special -> 3
Psyduck foloseste abilitate2 -> 2

Runda 4
Pikachu HP [33]
CDs: Abilitate1 [3] / Abilitate2 [3]
Stunned

Psyduck HP [30]
CDs: Abilitate1 [2] / Abilitate2 [5]

Pikachu este stunned
Psyduck foloseste atac normal -> 3

Runda 5
Pikachu HP [30]
CDs: Abilitate1 [2] / Abilitate2 [2]

Psyduck HP [30]
CDs: Abilitate1 [1] / Abilitate2 [4]

Pikachu foloseste atac special -> 3
Psyduck foloseste atac normal -> 3

Runda 6
Pikachu HP [27]
CDs: Abilitate1 [1] / Abilitate2 [1]

Psyduck HP [27]
CDs: Abilitate1 [0] / Abilitate2 [3]

Pikachu foloseste atac special -> 3
Psyduck foloseste abilitate1 -> 2

Runda 7
Pikachu HP [25]
CDs: Abilitate1 [0] / Abilitate2 [0]

Psyduck HP [24]
CDs: Abilitate1 [4] / Abilitate2 [2]

Pikachu foloseste atac special -> 3
Psyduck foloseste atac normal -> 3

Runda 8
Pikachu HP [22]
CDs: Abilitate1 [0] / Abilitate2 [0]

Psyduck HP [21]
CDs: Abilitate1 [3] / Abilitate2 [1]

Pikachu foloseste abilitate2 -> 4
Psyduck foloseste atac normal -> 3

Runda 9
Pikachu HP [22]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Psyduck HP [17]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Psyduck este stunned
Pikachu foloseste atac special -> 3

Runda 10
Pikachu HP [22]
CDs: Abilitate1 [0] / Abilitate2 [4]

Psyduck HP [14]
CDs: Abilitate1 [1] / Abilitate2 [0]

Pikachu foloseste abilitate1 -> 6
Psyduck foloseste abilitate2 -> 2

Runda 11
Pikachu HP [20]
CDs: Abilitate1 [4] / Abilitate2 [3]
Stunned

Psyduck HP [8]
CDs: Abilitate1 [0] / Abilitate2 [5]

Pikachu este stunned
Psyduck foloseste abilitate1 -> 2

Runda 12
Pikachu HP [18]
CDs: Abilitate1 [3] / Abilitate2 [2]

Psyduck HP [8]
CDs: Abilitate1 [4] / Abilitate2 [4]

Pikachu foloseste atac special -> 3
Psyduck foloseste atac normal -> 3

Runda 13
Pikachu HP [15]
CDs: Abilitate1 [2] / Abilitate2 [1]

Psyduck HP [5]
CDs: Abilitate1 [3] / Abilitate2 [3]

Pikachu foloseste atac special -> 3
Psyduck foloseste atac normal -> 3

Runda 14
Pikachu HP [12]
CDs: Abilitate1 [1] / Abilitate2 [0]

Psyduck HP [2]
CDs: Abilitate1 [2] / Abilitate2 [2]

Pikachu foloseste abilitate2 -> 4
Psyduck foloseste atac normal -> 3

---------> A castigat Pikachu <---------
Pikachu {HP[35], AS[4], Def[2], SDef[3]}
				  V
Pikachu {HP[36], AS[5], Def[3], SDef[4]}


------ Se lupta Meowth cu Bulbasaur ------
Meowth {HP[42], AN[4], Def[5], SDef[3]}
Bulbasaur {HP[43], AS[6], Def[4], SDef[2]}
--- Caracteristici cu obiecte echipate ---
Meowth {HP[54], AN[6], Def[5], SDef[6]}
Bulbasaur {HP[43], AS[9], Def[5], SDef[2]}

Runda 1
Meowth HP [42]
CDs: Abilitate1 [0] / Abilitate2 [0]

Bulbasaur HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Meowth foloseste abilitate2 -> 1
Bulbasaur foloseste atac special -> 3

Runda 2
Meowth HP [42]
CDs: Abilitate1 [0] / Abilitate2 [3]
Dodged 

Bulbasaur HP [42]
CDs: Abilitate1 [0] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste abilitate1 -> 6

Runda 3
Meowth HP [36]
CDs: Abilitate1 [0] / Abilitate2 [2]

Bulbasaur HP [41]
CDs: Abilitate1 [4] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste abilitate2 -> 5

Runda 4
Meowth HP [31]
CDs: Abilitate1 [0] / Abilitate2 [1]

Bulbasaur HP [40]
CDs: Abilitate1 [3] / Abilitate2 [3]

Meowth foloseste abilitate1 -> 5
Bulbasaur foloseste atac special -> 3

Runda 5
Meowth HP [31]
CDs: Abilitate1 [4] / Abilitate2 [0]
Dodged 

Bulbasaur HP [35]
CDs: Abilitate1 [2] / Abilitate2 [2]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

Runda 6
Meowth HP [28]
CDs: Abilitate1 [3] / Abilitate2 [0]

Bulbasaur HP [34]
CDs: Abilitate1 [1] / Abilitate2 [1]

Meowth foloseste abilitate2 -> 1
Bulbasaur foloseste atac special -> 3

Runda 7
Meowth HP [28]
CDs: Abilitate1 [2] / Abilitate2 [3]
Dodged 

Bulbasaur HP [33]
CDs: Abilitate1 [0] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste abilitate1 -> 6

Runda 8
Meowth HP [22]
CDs: Abilitate1 [1] / Abilitate2 [2]

Bulbasaur HP [32]
CDs: Abilitate1 [4] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

Runda 9
Meowth HP [19]
CDs: Abilitate1 [0] / Abilitate2 [1]

Bulbasaur HP [31]
CDs: Abilitate1 [3] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

Runda 10
Meowth HP [16]
CDs: Abilitate1 [0] / Abilitate2 [0]

Bulbasaur HP [30]
CDs: Abilitate1 [2] / Abilitate2 [0]

Meowth foloseste abilitate1 -> 5
Bulbasaur foloseste abilitate2 -> 5

Runda 11
Meowth HP [16]
CDs: Abilitate1 [4] / Abilitate2 [0]
Dodged 

Bulbasaur HP [25]
CDs: Abilitate1 [1] / Abilitate2 [3]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

Runda 12
Meowth HP [13]
CDs: Abilitate1 [3] / Abilitate2 [0]

Bulbasaur HP [24]
CDs: Abilitate1 [0] / Abilitate2 [2]

Meowth foloseste abilitate2 -> 1
Bulbasaur foloseste abilitate1 -> 6

Runda 13
Meowth HP [13]
CDs: Abilitate1 [2] / Abilitate2 [3]
Dodged 

Bulbasaur HP [23]
CDs: Abilitate1 [4] / Abilitate2 [1]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

Runda 14
Meowth HP [10]
CDs: Abilitate1 [1] / Abilitate2 [2]

Bulbasaur HP [22]
CDs: Abilitate1 [3] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste abilitate2 -> 5

Runda 15
Meowth HP [5]
CDs: Abilitate1 [0] / Abilitate2 [1]

Bulbasaur HP [21]
CDs: Abilitate1 [2] / Abilitate2 [3]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

Runda 16
Meowth HP [2]
CDs: Abilitate1 [0] / Abilitate2 [0]

Bulbasaur HP [20]
CDs: Abilitate1 [1] / Abilitate2 [2]

Meowth foloseste abilitate1 -> 5
Bulbasaur foloseste atac special -> 3

Runda 17
Meowth HP [2]
CDs: Abilitate1 [4] / Abilitate2 [0]
Dodged 

Bulbasaur HP [15]
CDs: Abilitate1 [0] / Abilitate2 [1]

Meowth foloseste abilitate2 -> 1
Bulbasaur foloseste abilitate1 -> 6

Runda 18
Meowth HP [2]
CDs: Abilitate1 [3] / Abilitate2 [3]
Dodged 

Bulbasaur HP [14]
CDs: Abilitate1 [4] / Abilitate2 [0]

Meowth foloseste atac normal -> 1
Bulbasaur foloseste atac special -> 3

---------> A castigat Bulbasaur <---------
Bulbasaur {HP[43], AS[6], Def[4], SDef[2]}
				  V
Bulbasaur {HP[44], AS[7], Def[5], SDef[3]}

############################################
##############  Lupta finala  ##############
############################################

------ Se lupta Squirtle cu Squirtle ------
Squirtle {HP[61], AS[4], Def[6], SDef[6]}
Squirtle {HP[60], AS[3], Def[5], SDef[5]}
--- Caracteristici cu obiecte echipate ---
Squirtle {HP[63], AS[9], Def[8], SDef[8]}
Squirtle {HP[70], AS[3], Def[6], SDef[5]}

Runda 1
Squirtle HP [61]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [60]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 2
Squirtle HP [61]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle HP [58]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste atac special -> 4

Runda 3
Squirtle HP [61]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle HP [54]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste abilitate1 -> 4
Squirtle foloseste abilitate2 -> 2

Runda 4
Squirtle HP [59]
CDs: Abilitate1 [3] / Abilitate2 [0]
Stunned

Squirtle HP [50]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste abilitate1 -> 4

Runda 5
Squirtle HP [55]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle HP [50]
CDs: Abilitate1 [3] / Abilitate2 [1]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 6
Squirtle HP [55]
CDs: Abilitate1 [1] / Abilitate2 [2]

Squirtle HP [48]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste atac special -> 4

Runda 7
Squirtle HP [55]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle HP [44]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle foloseste abilitate1 -> 4
Squirtle foloseste atac special -> 0

Runda 8
Squirtle HP [55]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle HP [40]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste atac special -> 4
Squirtle foloseste abilitate1 -> 4

Runda 9
Squirtle HP [51]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle HP [36]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 10
Squirtle HP [51]
CDs: Abilitate1 [1] / Abilitate2 [2]

Squirtle HP [34]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste atac special -> 4

Runda 11
Squirtle HP [51]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle HP [30]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle foloseste abilitate1 -> 4
Squirtle foloseste atac special -> 0

Runda 12
Squirtle HP [51]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle HP [26]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste abilitate2 -> 2

Runda 13
Squirtle HP [49]
CDs: Abilitate1 [2] / Abilitate2 [2]
Stunned

Squirtle HP [24]
CDs: Abilitate1 [0] / Abilitate2 [2]
Stunned

Squirtle este stunned
Squirtle este stunned

Runda 14
Squirtle HP [49]
CDs: Abilitate1 [1] / Abilitate2 [1]

Squirtle HP [24]
CDs: Abilitate1 [0] / Abilitate2 [1]

Squirtle foloseste atac special -> 4
Squirtle foloseste abilitate1 -> 4

Runda 15
Squirtle HP [45]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [20]
CDs: Abilitate1 [3] / Abilitate2 [0]

Squirtle foloseste atac special -> 4
Squirtle foloseste abilitate2 -> 2

Runda 16
Squirtle HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Squirtle HP [16]
CDs: Abilitate1 [2] / Abilitate2 [2]

Squirtle este stunned
Squirtle foloseste atac special -> 0

Runda 17
Squirtle HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle HP [16]
CDs: Abilitate1 [1] / Abilitate2 [1]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

Runda 18
Squirtle HP [43]
CDs: Abilitate1 [0] / Abilitate2 [2]

Squirtle HP [14]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Squirtle este stunned
Squirtle foloseste abilitate1 -> 4

Runda 19
Squirtle HP [43]
CDs: Abilitate1 [3] / Abilitate2 [1]

Squirtle HP [10]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste atac special -> 4
Squirtle foloseste atac special -> 0

Runda 20
Squirtle HP [43]
CDs: Abilitate1 [2] / Abilitate2 [0]

Squirtle HP [6]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste atac special -> 4
Squirtle foloseste atac special -> 0

Runda 21
Squirtle HP [43]
CDs: Abilitate1 [1] / Abilitate2 [0]

Squirtle HP [2]
CDs: Abilitate1 [0] / Abilitate2 [0]

Squirtle foloseste abilitate2 -> 2
Squirtle foloseste atac special -> 0

---------> A castigat Squirtle <---------
Squirtle {HP[61], AS[4], Def[6], SDef[6]}
				  V
Squirtle {HP[62], AS[5], Def[7], SDef[7]}

##  Marcel a iesit castigator din arena  ##
